pipelines/batch/setup_eval_job.py

```diff
@@ -21,6 +22,7 @@ def main(
     job_id: str,
     pool_id: str,
     diseases: str,
+    output_subdir: str | Path = "./",
```

Comment:
Why is the output directory not configurable? And if it were, shouldn't we just have one arg where you can put the full path?

Reply:
See https://github.com/CDCgov/pyrenew-hew/pull/159/files/a2fa9afa9651fda572c50b4a2db90b8379dc468b

Because this is just an alias for the root of the output blob storage container, I don't think it needs a configurable name.